internal/common/roles/speaker.go

```diff
@@ -6,6 +6,7 @@ import "github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 // optionally implemented by individual islands.
 type Speaker interface {
 	PayJudge() shared.SpeakerReturnContent
+	GetJudgeSalary() shared.Resources
```

Comment:
Does this seem like it should be a utility function in rules more than it should be a function they have to override? We should not change the interface I feel

Reply:
Yeah that's just cuz i made the function first and later realised that's where it belongs 😛 
I can merge the second function into the first, gimme a sec